internal/operators/ocs/test_utils.go

```diff
@@ -0,0 +1,203 @@
+package ocs
+
+import (
+	"encoding/json"
+
+	. "github.com/onsi/gomega"
+	"github.com/openshift/assisted-service/models"
+)
+
+func DefaultInventory() string {
+	inventory := models.Inventory{
+		Interfaces: []*models.Interface{
+			{
+				Name: "eth0",
+				IPV4Addresses: []string{
+					"1.2.3.4/24",
+				},
+				IPV6Addresses: []string{
+					"1001:db8::10/120",
+				},
+			},
+		},
+		CPU: &models.CPU{
+			Count: 16,
+		},
+		Memory: &models.Memory{
+			UsableBytes: 64000000000,
+		},
+		Disks: []*models.Disk{
+			{
+				SizeBytes: 20000000000,
+			}, {
+				SizeBytes: 40000000000,
+			},
+		},
+	}
+	b, err := json.Marshal(&inventory)
+	Expect(err).To(Not(HaveOccurred()))
+	return string(b)
+}
+
+func InventoryWithTwoDisks(cpus int64, ram int64) string {
```

Comment:
You might want to abstract this code, you seem to be reusing most of its body in these functions, except for the Disk part.